openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -658,9 +661,12 @@ def _get_arnold_render_payload(self, data):
         # Plugin Info
         ass_file, _ = os.path.splitext(data["output_filename_0"])
         ass_filepath = ass_file + ".ass"
+        current_verbosity_level = cmds.getAttr(
```

Comment:
@moonyuet I think this was missed - this should have been removed and also get taken from settings, right?

Reply:
I have removed these duplicated attributes in the PR which has tagged you.